src/HTTP/Request.ts

```diff
@@ -65,6 +66,17 @@ export interface RetrievalPreferences {
 type ResolveCallback = ( response:Response ) => void;
 type RejectCallback = ( error:HTTPError ) => void;
 
+function __addErrors<T extends {}>( o:T ):void {
+	Object
+	.keys( o )
+	.map( k => o[ k ] )
+	.filter( e => {
```

Comment:
Do not use `.filter()` if the returned value is not used; use `.forEach()` instead.